books/free-programming-books-hu.md

```diff
@@ -24,7 +24,7 @@
 * [Az adatbázistervezés alapjai és titkai](http://mek.oszk.hu/11100/11123/index.phtml) - Halassy Béla (Word, PDF)
 * [Ember, információ, rendszer](http://mek.oszk.hu/11100/11122/index.phtml) - Halassy Béla (Word, PDF)
 * [Formális nyelvek](http://mek.oszk.hu/05000/05099/index.phtml) - Bach Iván (PDF)
-* [Mese a felhasználó központú tervezőről](http://mek.oszk.hu/11700/11748/index.phtml) - David Travis, ford.: Favorit Fordító Iroda (PDF)
```

Comment:
is this a translator notation

Reply:
yes i think, or the editorial

```
A fordítás alapja:
David Travis – The Fable of the User-Centred Designer
2009 USERFOCUS

Fordította: Favorit Fordító Iroda, 2012
```

Reply:
Recovered at 0a4dbc179e4b2f6346fc15579d7c5edec1b3433e